### ISSUE TITLE
On latest core, seems to break emoji extended

### ISSUE THREAD

**Issue by psychobunny** [Schamper/nodebb-plugin-poll#15]
https://github.com/frissdiegurke/nodebb-plugin-emoji-extended/ stops working for some reason, I guess maybe something to do with the regexes involved. I know its probably weird to complain that a plugin by author A breaks a plugin by author B but I thought I'd put this up anyways :)

**Comment by TedRinehart** [comment]
Any follow up on this? The two seem to not coexist well on 0.6.0.

I had nodebb-plugin-poll@0.1.1 and nodebb-plugin-emoji-extended@0.4.1-3 installed. No errors were being thrown in the console of any sort, but I noticed that there was no Emoji Extended route under Installed Plugins as there had been previously, and emojis were not parsing on the board. After disabling poll and then manually installing 0.4.0-3 of emoji-extended, I was able to return the emoji parsing to user chats and posts.

I was then able to reactivate poll, and existing polls became functional once more.
